src/pages/solutions.js

```diff
@@ -49,7 +49,7 @@ const Solution = ({ title, image, link }) => (
           objectPosition="50% 50%"
           alt={title}
         />
-        <H4 textAlign="center">{title}</H4>
+        <Text as='h4' fontFamily='heading' textAlign="center">{title}</Text>
```

Comment:
Why cant we just make a custom component called H4 with Text as and font family.
There is too much repetitiveness